rest/data/interface.go

```diff
@@ -109,4 +118,11 @@ type BuildloggerOptions struct {
 	SoftSizeLimit int
 }
 
+type TestResultsOptions struct {
```

Comment:
Add documentation here

Reply:
what would be in the documentation? The BuildloggerOptions struct doesn't have any to base it off of

Reply:
Just say what is it for, something like "TestResultsOptions holds the options for test results related connector functions."

Reply:
Linters expect every function to be documented. The Evergreen code base predates the convention, so is woefully underdocumented, but many of our libraries are fully documented. If a package is documented, it becomes really nice to read its documentation in your editor or on the internet https://godoc.org/github.com/evergreen-ci/cedar/model.

Reply:
The linters don't actually check for doc comments on exported functions because it tends to be annoying and noisy. We can enable it if it's useful, but it's not on right now.

Reply:
Right, I cited linters not to say that we do or should use them, but as evidence for the convention.